azext_iot/digitaltwins/providers/model.py

```diff
@@ -16,7 +16,7 @@
 logger = get_logger(__name__)
 MAX_MODELS_API_LIMIT = 250
 # avagraw - Max number of models the API's dependency resolution can handle when models are created across multiple API calls.
-MAX_MODELS_PER_BATCH = 35
+MAX_MODELS_PER_BATCH = 30
```

Comment:
this discrepancy is very concerning because there is almost a 10 times difference between what the api promises and actually delivers

Reply:
These findings should be captured and discussed with ADT.

Reply:
yeah this is changed upon testing, and the time that the test run increased is around 3-4 seconds. I think the problem is not quite about itself at this point, it might because we sent the requests too fast and the model imported are becoming disordered😥

Reply:
Comment on the "...we sent the requests too fast and the model imported are becoming disordered" , _assuming_ you are talking about the service side processing.....I would argue the service should not return so quickly if processing the models hasn't been finished, IIRC this add models operation is intended to be synchronous.

Reply:
Also the client should be sending models in correct order, if that is not happening lets fix it.